pkg/cri/server/testing/fake_cni_plugin.go

```diff
@@ -43,6 +43,11 @@ func (f *FakeCNIPlugin) Remove(ctx context.Context, id, path string, opts ...cni
 	return nil
 }
 
+// Check the network of PodSandbox.
+func (f *FakeCNIPlugin) Check(ctx context.Context, id, path string, opts ...cni.NamespaceOpts) error {
```

Comment:
I implemented the Check method in go-cni, I am sorry this appears to have broken a few things! I will be more diligent next time! Always feel free to reach out to me on slack, I will respond fast!

Reply:
No problem at all, updating a test mock is no big deal. We don't usually freeze all our libraries right before release since it is rare we need to go back through with late updates. As a contributor, you did the right thing, we just ask for patience if we are taking longer to merge something because of a pre-release soft freeze.